concent_api/core/message_handlers.py

```diff
@@ -1286,7 +1286,7 @@ def handle_send_subtask_results_verify(
 
     verification_deadline = subtask_results_rejected.timestamp + settings.ADDITIONAL_VERIFICATION_CALL_TIME
 
-    if not subtask_results_rejected.timestamp < get_current_utc_timestamp() <= verification_deadline:
+    if not get_current_utc_timestamp() <= verification_deadline:
```

Comment:
is there an issue for this?

Reply:
@pawelkisielewicz I think you are mixing two issues here, if yes, add second one to the description.

Reply:
done

Reply:
@pawelkisielewicz Github will not catch it this way. You need to write
```
Resolves #532
Resolves #538
```